about.html

```diff
@@ -4,36 +4,101 @@
+          The Global Summer springs brings together the community of experts, academics, and developer to join together and share the latest technologies that are in the market.
+        </h4>
+      </article>
+      <article class="header_article_desktop">
+        <h4>
+          The Global Summer springs brings together the community of experts, academics, and developer to join together and share the latest technologies that are in the market.
+          This year, we hope to expand our invitation list including organizations and individuals who want to work with us on shared projects that advance the cause of the Commons,
+          ways to learn and apply this new tecnologies.
+          So if youre active and engaged in the worlds of web development - free software advocates, Wikipedians, Open Knowledge, galleries, workshops, web designs, inversionists,
+          stake holders, founders - we hope you'll consider joining us this year to build a stronger, more vibrant better coding practices, better desings and technologies.
+        </h4>
+      </article>
+      <div class="last_header_part">
+        <p>Please contact us per Email for any further questions about CREATIVE ARMANDO SUMMER 2015!</p>
+        <a class="link_last_header" href="mailto:businesswitharmando@gmail.com">businesswitharmando@gmail.com</a>
+      </div>
+    </header>
+    <section class="miidle_section">
+      <h2>CREATIVE ARMANDO SUMMER 2015 Logo</h2>
+      <hr class="division_logo">
+      <p class="paragraph_mobile">CREATIVE ARMANDO SUMMER 2022 logo was decided through the logo conference from 8. June to 7. July</p>
+      <p class="paragraph_desktop">The logo of CC Global SUMMER 2022 was decided through the logo competition from 8. June to 7. July.
+        After online public voting and Selection Committees judging process, the logo from Naresh Agrawal won the prize.
+        After consultation with Mr. Agrawal we optimized the prize-winning-logo for this years summit as following:
+      </p>
+      <div class="global_icon"></div>
+      <div class="global_icon_mobile"></div>
+      <hr class="division_middle">
+    </section>
+    <section class="last_section">
+      <h2>See the past CREATIVE ARMANDO SUMMER 2015 Logo</h2>
+      <hr class="division_logo">
+      <p class="para_last_section">Take a look at the last two CREATIVE ARMANDO SUMMER 2015 Logo which took place in Buenos Aires and in Warsaw.</p>
+      <div class="container_past_img">
+        <div class="global_2013"></div>
+        <div class="global_2011"></div>
+      </div>
+      <div class="container_past_img_mobile">
+        <div class="global_2013_mobile"></div>
+        <div class="global_2011_mobile"></div>
+      </div>
+    </section>
+    <section class="section_partners">
+      <div class="icon_partners"></div>
+    </section>
+    <footer class="footer_section">
+      <div class="cc_logo_last"></div>
+      <div class="commons">
+        creative <br> commons Mexico
+      </div>
```

Comment:
- [ ] In the mobile section of your About page, kindly make the footer match that of the template design. There is a logo aligned to the left and the copyright aligned to the right. It is not divided into three sections as you have. Also, kindly correct the typographical error to **_2022_**.

### Your Design

![image](https://user-images.githubusercontent.com/77106962/181381500-5148321e-b5d1-4b79-9593-ff36c16e61fe.png)

### Design Template

![image](https://user-images.githubusercontent.com/77106962/181381643-4ac2665f-4f1e-4b18-ab99-a3bee7201104.png)